rasa/core/training/story_reader/yaml_story_reader.py

```diff
@@ -79,118 +100,147 @@ def read_from_parsed_yaml(
         ):
             return []
 
-        stories = parsed_content.get(KEY_STORIES, [])
-        self._parse_data(stories, is_rule_data=False)
+        for key, parser_class in {
+            KEY_STORIES: StoryParser,
+            KEY_RULES: RuleParser,
+        }.items():
+            stories = parsed_content.get(key, [])
```

Comment:
should we rename this var as it might be rules? maybe
```suggestion
            data = parsed_content.get(key, [])
```

Reply:
🙈